packages/block-editor/src/components/border-radius-control/test/utils.js

```diff
@@ -180,3 +182,72 @@ describe( 'mode', () => {
 		expect( mode( values ) ).toBe( 'a' );
 	} );
 } );
+
+describe( 'getPresetValueFromCustomValue', () => {
+	const presets = [
+		{ name: 'None', slug: '0', size: '0' },
```

Comment:
Nit: I think we might want this test preset to mirror what is actually [added in the `BorderRadiusControl`](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/border-radius-control/index.js#L34), i.e. a numeric `0`.

```
{ name: __( 'None' ), slug: '0', size: 0 }
```

Reply:
Got it, I will fix this